engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkRestManager.scala

```diff
@@ -86,7 +86,9 @@ class FlinkRestManager(config: FlinkConfig, modelData: ModelData, mainClassName:
   //NOTE: Flink <1.10 compatibility - protected to make it easier to work with Flink 1.9, JobStatus changed package, so we use String in case class
   protected def mapJobStatus(overview: JobOverview): StateStatus = {
     toJobStatus(overview) match {
-      case JobStatus.RUNNING => FlinkStateStatus.Running
+      case JobStatus.RUNNING if overview.tasks.running == overview.tasks.total => FlinkStateStatus.Running
```

Comment:
I think we should count also finished as running? Also, for batch jobs there may be quirks around scheduling tasks lazily

Reply:
I wonder what is the case for streaming that some task is finished? But "finished" also doesn't look like "during deploy" so maybe it is a good way to handle that - changed.
Regards to second one I've extracted protected methods which can be overridden in the same way as it was before change. Is it ok for now or sth need to be changed now in `PeriodicDeploymentManager`?